lib/rules/max-attributes-per-line.js

```diff
@@ -128,16 +129,45 @@ module.exports = {
       return defaults
     }
 
-    function showErrors (attributes, node) {
+    function getPropData (prop) {
+      let propType = 'Attribute'
+      let propName = prop.key.name
+
+      if (utils.isBindingAttribute(prop)) {
+        propType = 'Binding'
+        propName = prop.key.raw.argument
```

Comment:
argument can be null when `v-bind="$props"` and error message is going to look:
```
Binding "null" should be on a new line.
```

```suggestion
        propName = prop.key.raw.argument || 'object as props'
```

see. https://vuejs.org/v2/guide/components-props.html#Passing-the-Properties-of-an-Object

Reply:
Good catch! I forgot about this case. I think it should actually report `Directive "bind" should be on a new line`

Reply:
thank you 👍